service/builder.go

```diff
@@ -22,31 +22,25 @@ package service
 
 import "github.com/pkg/errors"
 
+// WithModules is a helper to create a service without any options
+func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
Agree that it's confusing, but we will give people (at uber) a templated app that calls .Build() so hopefully they won't even notice, they just get a `(Service, error)`